### PR TITLE
introduce python auto formatting

### PR DESCRIPTION
**Pull request by XzzX**
Any suggestions for a PEP8 config file?

### CODE REVIEW

File: format.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #! /bin/bash
 
 git ls-files -- src*.cpp src*.hpp testsuite*.cpp testsuite*.hpp | xargs clang-format -i -style=file
+git ls-files -- src*.py testsuite*.py | xargs autopep8 --in-place --aggressive --aggressive
```
Comment (jnvance) [inline format.sh:4]: Maybe also include `examples*.py`, or should it be done after #369?

**Comment by jkrajniak** [comment]
> Any suggestions for a PEP8 config file?

yes, let's not stick to 80 characters, I think it is still in the standard but it's really annoying and old - 100, 120 seems reasonable, IMO.

**Comment by XzzX** [comment]
> > Any suggestions for a PEP8 config file?
> 
> yes, let's not stick to 80 characters, I think it is still in the standard but it's really annoying and old - 100, 120 seems reasonable, IMO.

Good point, should be the same as for C++.

File: format.sh (hunk 1 above)
Comment (XzzX) [reply format.sh:4]: I think both are independent. I will include examples.

**Comment by XzzX** [comment]
We have circular imports...

**Pull request closed without merging**